src/components/Toolbar.ts

```diff
@@ -74,6 +81,52 @@ const toolbarView = (vnode: Vnode<IToolbar>) => {
           },
           "Import HAR Session"
         ),
+        m(
+          "a.button.is-outlined.is-small.control",
+          {
+            class: ngrokStreamLock === -1 ? "ngrok-play" : "ngrok-stop"
+          },
+          m("hi", {
+            onclick: () => {
+            if (ngrokStreamLock != -1) {
+              console.log('stopping ngrok stream');
+              ngrokStreamLock = -1;
+              return;
+            }
+
+            // this permissions things works but it needs a better way
+            // to test and reset it!
+
+            var API_HOST = 'http://localhost:4040/'; // TODO: make this customisable in settings
+
+            requestPerms(API_HOST).then(() => {
+              let timeoutId = window.setTimeout(function pollStream() {
+                console.log('requesting new data...');
+                let newEntries = fetch(`${API_HOST}api/requests/http`, {
```

Comment:
This seems unused and only holds void, you don't need the `newEntries` variable.